patchnetvlad/configs/train.ini

```diff
@@ -0,0 +1,36 @@
+[train]
+batchsize = 4
+cachebatchsize = 20
+optim = SGD
+seed = 123
+lr = 0.0001
+lrstep = 5
+lrgamma = 0.5
+weightdecay = 0.001
+momentum = 0.9
+patience = 10
+evalevery = 1
+margin = 0.1
+num_clusters = 16
+includepanos = False
+imageresizew = 640
+imageresizeh = 480
+
+[val]
```

Comment:
Where are those values being used? (all the ones in the `[val]` group)

Reply:
Actually they are not being used anywhere, I'll delete the [val] group